content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,115 @@
+However, exhaustive KPI (i.e. Number of requests, ratio of errors, number of visitors...) should be computed upon all logs (including the 200 response code logs).
+
+This guide uses Logging without Limits&trade; to identify a noisy logging service and status, exclude irrelevant status logs, and set custom metrics from the excluded logs to continue to track KPI over time.
+
+## 1. Identify a noisy logging service and status
+
+Your noisiest logging service contains several logs, some of which may be irrelevant for troubleshooting. By identifying your noisiest logging service, you can quickly find the information you need to filter irrelevant logs by status.
+
+{{< img src="logs/guide/logging-without-limits.gif" alt="Log Patterns View" responsive="true" style="width:100%;">}}
+
+**To identify your noisiest logging service and status**:
+
+1. In Log Explorer, select graph view located next to the search bar. 
+2. Below the search bar, set `count` * group by `service` and `limit` to top 10.
+3. Select `Top List` from the dropdown menu next to `hide controls`.
+4. Click on the first listed service and select `search for` in the populated menu. This generates a search, which is visible in the search bar above, based on your `service` facet.
+5. Switch group by `service` to group by `status`. This generates a top statuses list for your service.
+6. Click on the first listed status and select `search for` in the populated menu. This adds your `status` facet to the search.
+
+**Note**: These steps are applicable to any high volume logging query to generate a top list. You can group by any facet, such as `host` or `client name` versus `service` or `status`.
+
+## 2. Identify noisy logs with log patterns
+Switch to the [patterns][2] view, located next to the graph view, to view the patterns associated with your service and status.
+
+{{< img src="logs/guide/patterns.png" alt="Log Patterns View" responsive="true" style="width:100%;">}}
+
+The patterns view is helpful when identifying and filtering noisy patterns. It shows the number of logs matching a pattern, split by service and status. Click on the first pattern to view a detailed log of events relating to your status. A contextual panel will populate with information about your noisiest status pattern.
+
+## 3. Exclude noisy logs with an exclusion filter
+
+The selected pattern contextual panel lists every instance (event) of a log pattern and creates a custom search query based on the selected pattern. Use this query in an exclusion filter to remove those logs from your index. Note that excluded logs are still available in the [live tail view][3] and can be [sent to log archives][4], or used to [generate metrics][5].
+
+**To create an exclusion filter for your top log pattern**:
+
+{{< img src="logs/guide/exclusion-filter.png" alt="Log Patterns View" responsive="true" style="width:100%;">}}
```

Comment:
The image should go after the list of instructions i think, since it represents the final result of those instructions.